corehq/apps/domain/views/internal.py

```diff
@@ -99,6 +104,8 @@ def internal_settings_form(self):
             'auto_case_update_limit': self.domain_object.auto_case_update_limit,
             'use_custom_odata_feed_limit': 'Y' if self.domain_object.odata_feed_limit else 'N',
             'odata_feed_limit': self.domain_object.odata_feed_limit,
+            'use_custom_gsheet_limit': 'Y' if gsheet_limit else 'N',
+            'gsheet_limit': gsheet_limit.limit_value if gsheet_limit else 20,
```

Comment:
The 20 here is hardcoded because we need a get_default_limit function that is being introduced in a different branch. I will update this code in that branch.